lib/currency-utils.js

```diff
@@ -46,13 +31,22 @@ export function formatCurrency(amount, currency = 'USD', options = {}) {
     minimumFractionDigits = options.precision;
     maximumFractionDigits = options.precision;
   }
-  return amount.toLocaleString(getLocaleFromCurrency(currency), {
+  return amount.toLocaleString(options.locale, {
     style: 'currency',
     currency,
     minimumFractionDigits: minimumFractionDigits,
     maximumFractionDigits: maximumFractionDigits,
+    currencyDisplay: 'narrowSymbol', // We manually add the exact currency (e.g. "$10 USD") in many places. This is to prevent showing the currency twice is some locales (10$US USD)
```

Comment:
This is the bit that fixes https://github.com/opencollective/opencollective/issues/3898